src/openapi/components/schemas/ChangemakerFieldValue.json

```diff
@@ -0,0 +1,66 @@
+{
+	"type": "object",
+	"properties": {
+		"id": {
+			"type": "integer",
+			"readOnly": true,
+			"example": 1
+		},
+		"changemakerId": {
+			"type": "integer",
+			"readOnly": true,
```

Comment:
I think this `readOnly` flag (or something else prevents it from showing up in the OpenAPI example schema on post:

<img width="660" height="309" alt="image" src="https://github.com/user-attachments/assets/bb462e54-4e5e-439d-847c-539840d13cbe" />